### DISCUSSION TITLE
漫游代理兼容biliroaming-ts-server-vercel

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#790] [General]
https://github.com/bili-vd-bak/biliroaming-ts-server-vercel

**Comment by ManyullynSword** [comment]
请问这是否和我之前提出的讨论有关？或是说是其它的问题或功能？

**Reply by ywmoyue** [reply]
可能有关，这个是看其他人提出用这个报错412但是安卓上bili-roaming正常

**Comment by ywmoyue** [comment]
@ManyullynSword 我根据这个项目的readme部署后没改任何配置测试没问题，你也试试？需要有一个域名

**Reply by ywmoyue** [reply]
https://github.com/bili-vd-bak/biliroaming-ts-server-vercel/issues/17 看这个issue使用其他代理服务报错412的使用这个仍然报错412

可能跟账号有关，我再看看安卓端的请求有什么区别 🤔

**Comment by ywmoyue** [comment]
@Re-hd 如果要使用自行部署的代理服务的话可以试试这个项目
